### PR TITLE
applications: machine_learning: Update documentation

### PR DESCRIPTION
**Pull request by maje-emb**
Added the nRF54H20 board description to the documentation.

### CODE REVIEW

File: applications/machine_learning/app_desc.rst [hunk 1]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   **LED2** turns on for some time.
+#. Shake the device.
+   The machine learning model detects an anomaly.
+   **LED1** and **LED2** turn on for some time.
+#. Gesture detection results can also be read using the Nordic Status Message Service.
+   Start the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+#. Connect to the device from the application.
+   The device is advertising as ``Sensor HUB``.
+   The services of the connected device are shown.
+   If the device can not be found - the PDK might be connected to the other device.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:555]:
```suggestion
   If the device cannot be found - the PDK might be connected to the other device.
```

File: applications/machine_learning/app_desc.rst [hunk 2]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   Terminate the connection with the other device, refresh the scanning and connect to the PDK.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:556]:
```suggestion
   Terminate the connection with the other device, refresh the scanning, and connect to the PDK.
```

File: applications/machine_learning/app_desc.rst [hunk 3]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   If the device disconnects shortly after being connected - the PDK might have saved bonding data from previous connection.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:557]:
```suggestion
   If the device disconnects shortly after being connected, the PDK might have saved bonding data from previous connection.
```

File: applications/machine_learning/app_desc.rst [hunk 4]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   The bonding data can be erased if you double-click the **Button 0** within 3 seconds after the application reboots.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:558]:
```suggestion
   You can erase the bonding data by double-clicking the **Button 0** within three seconds after the application reboots.
```

File: applications/machine_learning/app_desc.rst [hunk 5]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+#. Find **Nordic Status Message Service** by its name or UUID (57a70001-9350-11ed-a1eb-0242ac120002).
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:559]:
```suggestion
#. Find **Nordic Status Message Service** by its name or UUID (``57a70001-9350-11ed-a1eb-0242ac120002``).
```

File: applications/machine_learning/app_desc.rst [hunk 6]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   The ``updown`` gesture is recognized by the machine learning model and **Nordic Status Message Service** is updated.
+   **LED0** turns on for some time.
+#. Rotate the device.
+   The ``rotate`` gesture is recognized by the machine learning model and **Nordic Status Message Service** is updated.
+   **LED1** turns on for some time.
+#. Tap the device.
+   The ``tap`` gesture is recognized by the machine learning model and **Nordic Status Message Service** is updated.
+   **LED2** turns on for some time.
+#. Disconnect the `nRF Connect for Mobile`_ application from the device.
+#. Press and hold **Button 0** for more than 5 seconds to switch to the data forwarding mode.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:574]:
```suggestion
#. Press and hold **Button 0** for more than five seconds to switch to the data forwarding mode.
```

File: applications/machine_learning/app_desc.rst [hunk 7]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   After the mode is switched, **LED0**, **LED1**, and **LED2** start blinking very slowly.
+#. You can test the data forwarding mode of the application with the `nRF Connect for Mobile`_.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:576]:
```suggestion
#. Test the data forwarding mode of the application with the `nRF Connect for Mobile`_:
```

File: applications/machine_learning/app_desc.rst [hunk 8]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+
+   a. Open the `nRF Connect for Mobile`_ application and connect to the device again.
+   #. Bond with the device from the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+   #. Find **Nordic UART Service** and enable notification of its "TX Charcteristic".
+   #. Observe the sensor readouts represented as comma-separated values.
+      Every new value represents a single sensor readout.
+
+#. You can also test the data forwarding mode with use of additional DK.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:584]:
```suggestion
#. Test the data forwarding mode with an additional DK:
```

File: applications/machine_learning/app_desc.rst [hunk 9]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+
+   a. Program the :ref:`central_uart` sample to a compatible development kit, for example the nRF52840 Development Kit.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:586]:
```suggestion
   a. Program the :ref:`central_uart` sample to a compatible development kit, for example the nRF52840.
```

File: applications/machine_learning/app_desc.rst [hunk 10]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   #. Turn on the programmed device.
+      After a brief delay, the Bluetooth® connection between the sample and the PDK is established.
+      The nRF54H20 PDK forwards the sensor readouts over NUS to the Central UART sample.
+      The **LED0**, **LED1**, and **LED2** on the nRF54H20 PDK start to blink rapidly.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:590]:
```suggestion
      The **LED0**, **LED1**, and **LED2** on the nRF54H20 PDK start blinking rapidly.
```

File: applications/machine_learning/app_desc.rst [hunk 11]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
...
+   #. Connect to the Bluetooth® Central UART sample with a terminal emulator (for example, PuTTY).
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:591]:
```suggestion
   #. Connect to the Bluetooth Central UART sample with a terminal emulator (for example, PuTTY).
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/machine_learning/app_desc.rst [hunk 12]
```diff
@@ -34,6 +34,11 @@ By default, the following sensors are used by the application:
 * nRF5340 DK - Simulated sensor (:ref:`sensor_sim`).
   The simulated sensor generates predefined waves as acceleration.
   This development kit does not have a built-in accelerometer.
+* nRF54H20 - The DK does not have a built-in accelerometer.
+  The DK has two configurations:
+
+   * single-core - Accelerometer (``ADXL362``) connected to the DK with the :ref:`pca63566` shield.
+   * dual-core - Simulated sensor (:ref:`sensor_sim`).
```
[the remark this answers is not in the document]
Comment (peknis) [reply applications/machine_learning/app_desc.rst:41]: Link target pca63566 not found and the doc build fails. Where do you want to link to?

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by maje-emb** [comment]
I want to link to boards/shields/pca63566/doc/index.rst

**Comment by peknis** [comment]
> pca63566

Unfortunately, that is not part of our doc set. Try linking with :doc:`pca63566`. If it does not work, then we need to drop the link.

File: applications/machine_learning/app_desc.rst [hunk 13]
```diff
@@ -486,6 +527,74 @@ After programming the application, perform the following steps to test the nRF M
 Optionally, you can also connect to the device using `Edge Impulse's data forwarder`_ and forward data to `Edge Impulse studio`_ (after logging in).
 See `Forwarding data to Edge Impulse studio`_ for details.
 
+Testing with the nRF54H20 PDK
+-----------------------------
+
+After programming the application, perform the following steps to test the nRF Machine Learning application on the PDK:
+
+1. Turn on your PDK.
```
Comment (dchat-nordic) [inline applications/machine_learning/app_desc.rst:535]: Is it PDK or DK now?

Comment (peknis) [reply applications/machine_learning/app_desc.rst:535]: Good point, DK it is.

Comment (maje-emb) [reply applications/machine_learning/app_desc.rst:535]: It should be DK.

File: applications/machine_learning/app_desc.rst [hunk 14]
```diff
@@ -208,6 +227,10 @@ By default, the following buttons are used by the application:
   * **Button 1** switches between data forwarding and running a machine learning model.
   * **Button 3** changes the signal generated by the simulated sensor.
 
+* nRF54H20 DK:
```
[the remark this answers is not in the document]
Comment (maje-emb) [reply applications/machine_learning/app_desc.rst:230]: Good idea.

File: applications/machine_learning/app_desc.rst [hunk 15]
```diff
@@ -34,6 +34,11 @@ By default, the following sensors are used by the application:
 * nRF5340 DK - Simulated sensor (:ref:`sensor_sim`).
   The simulated sensor generates predefined waves as acceleration.
   This development kit does not have a built-in accelerometer.
+* nRF54H20 - The DK does not have a built-in accelerometer.
+  The DK has two configurations:
+
+   * Single-core - Accelerometer (``ADXL362``) connected to the DK with the :file:`nrf/boards/shields/pca63566/doc/index.rst` shield.
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:40]:
```suggestion
   * Single-core - Accelerometer (``ADXL362``) connected to the DK with the PCA63566 shield.
     See details in :file:`nrf/boards/shields/pca63566/doc/index.rst`.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: applications/machine_learning/app_desc.rst [hunk 16]
```diff
@@ -220,42 +247,67 @@ The application defines common LED effects for both the machine learning results
...
-  The **LED1** displays the application state, and the **LED2** displays the signal generated by the simulated sensor.
+   .. group-tab:: nRF54H20 DK
+      nRF54H20 DK uses monochromatic LEDs to display the application state.
+      **LED0**, **LED1** and **LED2** display the application state.
 
-  * If the device is returning the machine learning prediction results, the **LED1** blinks for a predefined number of times and then turns off for a period of time.
-    Then the sequence is repeated.
-    The machine learning result is represented by the number of blinks:
+      * If the device is returning the machine learning prediction results, a pattern of LEDs are turn on.
+         The machine learning result is represented by:
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:298]:
```suggestion
        The machine learning result is represented by:
```

File: applications/machine_learning/app_desc.rst [hunk 17]
```diff
@@ -220,42 +247,67 @@ The application defines common LED effects for both the machine learning results
...
+
+   .. group-tab:: nRF52840 and nRF5340 DKs
+
+      Both nRF5340 DK and nRF52840 DK use monochromatic LEDs to display the application state.
+      The **LED1** displays the application state, and the **LED2** displays the signal generated by the simulated sensor.
 
-  * If the device is returning the machine learning prediction results, the LED uses the following predefined colors:
+      * If the device is returning the machine learning prediction results, the **LED1** blinks for a predefined number of times and then turns off for a period of time.
+         Then the sequence is repeated.
+         The machine learning result is represented by the number of blinks:
```
Comment (peknis) [inline applications/machine_learning/app_desc.rst:279]:
```suggestion
        Then the sequence is repeated.
        The machine learning result is represented by the number of blinks:
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by dchat-nordic — APPROVED** [review]

**Pull request merged**